.ci/functional_test.sh

```diff
@@ -77,3 +77,22 @@ if [[ "$set_interface" != $ok_res || "$del_interface" != $ok_res ]]; then
     echo "WireGuard test failed!"
     exit 1
 fi
+
+echo "Testing Firewall Rules"
+fz_bin="/opt/firezone/embedded/service/firezone/bin/firezone"
+nft_bin="/opt/firezone/embedded/sbin/nft"
+ok_res=":ok"
+user_id="5" # Picking a high enough user_id so there is no overlap
+device="%{ip: \"10.0.0.1\", ip6: \"fd00::3:2:1\", user_id: $user_id}"
+rule="%{destination: \"10.0.0.2\", user_id: $user_id, action: :drop, port_type: nil, port_range: nil}"
+add_user=`sudo $fz_bin rpc "IO.inspect(FzWall.CLI.Live.add_user($user_id))"`
+add_device=`sudo $fz_bin rpc "IO.inspect(FzWall.CLI.Live.add_device($device))"`
+add_rule=`sudo $fz_bin rpc "IO.inspect(FzWall.CLI.Live.add_rule($rule))"`
+del_rule=`sudo $fz_bin rpc "IO.inspect(FzWall.CLI.Live.delete_rule($rule))"`
+del_device=`sudo $fz_bin rpc "IO.inspect(FzWall.CLI.Live.delete_device($device))"`
+del_user=`sudo $fz_bin rpc "IO.inspect(FzWall.CLI.Live.delete_user($user_id))"`
```

Comment:
I don't *think* you need sudo for these? You may have to run as the `firezone` user though?

Reply:
I tried to [login with `firezone` user](https://github.com/firezone/firezone/pull/890/files#diff-f9a68fcc0b8b49723aef33049e692a0a40e2c5ce27b8a7da8a039f29643e8017R83) in the CI but it needs a password, I'm not sure what's the best way to handle this in the CI

Reply:
Re-adding sudo for now to make sure the CI passes.